azkaban-common/src/main/java/azkaban/executor/ExecutionFlowDao.java

```diff
@@ -277,6 +291,18 @@ private void updateExecutableFlow(final ExecutableFlow flow, final EncodingType
     }
   }
 
+  private void updateExecutableFlowStatus(final ExecutableFlow flow)
+    throws ExecutorManagerException {
+    final String UPDATE_FLOW_STATUS = "UPDATE execution_flows SET status = ? where exec_id = ?";
```

Comment:
I believe we need to add update time as well.

Reply:
Yes!